src/pages/favorites/favorites.component.ts

```diff
@@ -164,6 +167,14 @@ export class FavoritesComponent {
   }
   deleteTrip(trip): void {
     _.remove(this.trips, {name: trip.name});
-    this.tripSvc.deleteTrip(trip);
+    this.faveTripSvc.deleteTrip(trip);
+  }
+  removeRoute(route): void {
+    _.remove(this.routes, {name: route.name});
```

Comment:
Ohhh. I saw `routeToRemove: Route`, and thought that `Route` would just be the name of the route. I was wrong. Great!